musonant/questioner
# Chore setup database 162881785

#### What does this PR do?
- Setup database
#### Description of Task to be completed?
- Install and setup postgres database
- Implement migrations
- Implement seedings
#### How should this be manually tested?
- After cloning repository, run `npm install`
- Run `npm run migration`
- Databases should be created
#### Any background context you want to provide?
- NA
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162881785
#### Screenshots (if appropriate)
- NA
#### Questions:`
- NA
